### ISSUE TITLE
After non basic characters, title understands a new word

### ISSUE THREAD

**Issue by kiketordera** [go-sanitize/sanitize#11]
Regarding the "title" functionality:

If the input is about letters from A to Z without any accent, the code will work without problems, but if we put any character that has any punctuation marks, the code will understand there is any word when there is not. 

Example:
```
type User struct {
	Name                 string           form:"fname" san:"max=25,trim,title,xss"` 
}
```
if in the form field in the HTML, we put "hernandez"  "espana" or  "garcia", it will work without a problem, but if we add the punctuation for the language, like this: "hernández"  "españa" or  "garcía", it will translate it into: "hernáNdez"  "españA" and  "garcíA". When you add a punctuation sign in the letter, it takes the next letter as the beginning of a word. In English is not a problem, but in Latin languages, such as Spanish, French, and so on, it makes it impossible to use this property of the framework.

**Comment by mattwelke** [comment]
@kiketordera  Thanks for the bug report. Are you using this library in production?

Looks like instead of using a library dedicated to title casing, we implemented this from scratch: https://github.com/go-sanitize/sanitize/blob/172e36923e5862a579ac0878e7ffd5a9a5e12f6b/string.go

Perhaps we should update this to delegate this to a dedicated library for title casing. Or we could fix our implementation. I'm generally too busy to update this library, but I can review PRs if someone submits one.

**Comment by kiketordera** [comment]
Sorry for the late respond. I am so busy also, but I found a very neat and easy solution. We could use strings.Title() function from go language itself, I tried and it does not have the same problems for latin languages, so I think this will be a nice fix to your library. I been inspired by this article: https://www.jeremymorgan.com/tutorials/go/learn-golang-casing/

You think this easy solution is good enough for implementing it?
